src/shared/wtypes.rs

```diff
@@ -95,6 +95,7 @@ pub type BSTR = *mut OLECHAR;
 pub type LPBSTR = *mut BSTR;
 pub type VARIANT_BOOL = c_short;
 pub const VARIANT_TRUE: VARIANT_BOOL = -1;
+pub const VARIANT_FALSE: VARIANT_BOOL = 0;
```

Comment:
`VARIANT_FALSE` is defined above `VARIANT_TRUE`.

Reply:
Would you like me to swap them? In `wtypes.h` `VARIANT_TRUE` is defined above `VARIANT_FALSE`

Reply:
Ah, never mind. I apparently looked in the wrong file. When I `rg`d for them I saw them in `um\portabledevicetypes.idl` which has them with false above true. Keep it the way it is.